lib/pltr/v2/selectors/ui.js

```diff
@@ -236,6 +241,43 @@ export const orientationSelector = createSelector(
   }
 )
 
+export const stickyHeaderCountSelector = createSelector(
+  pinnedPlotlinesSelector,
+  timelineViewIsStackedSelector,
+  hierarchyLevelCount,
+  selectedOrientationSelector,
+  (pinnedPlotlines, timelineViewIsStacked, hierarchyLevelCount, selectedOrientation) => {
+    if (selectedOrientation == 'vertical') {
+      return 1
+    }
+    if (timelineViewIsStacked) {
```

Comment:
I'm not sure this is right, so on the stacked view we don't pin plot lines?

Reply:
@Quiescent im not sure what's the computation with stackedview.
It is `hierarchyLevelCount + pinnedPlotlines + 1`?